core/templates/dev/head/components/activity_tiles_infinity_grid_directive.html

```diff
@@ -1,22 +1,33 @@
 <script type="text/ng-template" id="components/activityTilesInfinityGrid">
-  <div infinite-scroll="showMoreExplorations()" infinite-scroll-distance="1"
+  <div infinite-scroll="showMoreActivities()" infinite-scroll-distance="1"
        infinite-scroll-disabled="endOfPageIsReached || searchResultsAreLoading">
-    <exploration-summary-tile ng-repeat="exp in allExplorationsInOrder track by $index"
-                              exploration-id="exp.id"
-                              exploration-title="exp.title"
-                              last-updated-msec="exp.last_updated_msec"
-                              objective="exp.objective"
-                              category="exp.category"
-                              ratings="exp.ratings"
-                              thumbnail-icon-url="exp.thumbnail_icon_url"
-                              thumbnail-bg-color="exp.thumbnail_bg_color"
-                              num-views="exp.num_views"
-                              contributors-summary="exp.human_readable_contributors_summary"
-                              is-community-owned="exp.community_owned"
-                              is-community-editable="expl.community_editable"
-                              class="protractor-test-exp-summary-tile"
-                              style="display: inline-block;">
-    </exploration-summary-tile>
+    <div ng-repeat="activity in allActivitiesInOrder track by $index" style="display: inline-block;">
+      <collection-summary-tile ng-if="activity.activity_type=='collection'"
```

Comment:
Please make sure the conditional has expected spacing, e.g.:

``ng-if="activity.activity_type === 'collection'"``